jaeger/info.go

```diff
@@ -17,33 +16,28 @@ func getJaegerInfo(client http.Client, endpoint *url.URL) (*JaegerInfo, int, err
 		return nil, http.StatusNoContent, nil
 	}
 
-	externalUrl := jaegerConfig.URL
-	if externalUrl == "" {
-		return nil, http.StatusServiceUnavailable, errors.New("Jaeger URL is not set in Kiali configuration")
-	}
-
-	if jaegerConfig.InClusterURL == "" {
-		return nil, http.StatusServiceUnavailable, errors.New("Jaeger URL in cluster is not set in Kiali configuration")
-	}
-
-	u := endpoint
-	u.Path = path.Join(u.Path, "/api/services")
-
-	_, code, err := makeRequest(client, u.String(), nil)
-	if err != nil || code != 200 {
+	u, err := url.Parse(jaegerConfig.InClusterURL)
+	if err != nil {
 		integration = false
-		error = "Error with internal connection with Jaeger"
-		if err != nil {
-			error += ": " + err.Error()
+		error = "Error parsing in cluster url for Jaeger : " + err.Error()
+	} else {
+		u.Path = path.Join(u.Path, "/api/services")
+		_, code, err := makeRequest(client, u.String(), nil)
+		if err != nil || code != 200 {
+			integration = false
+			error = "Error with internal connection with Jaeger"
+			if err != nil {
+				error += ": " + err.Error()
+			}
 		}
 	}
 
 	info := &JaegerInfo{
-		URL:                externalUrl,
+		URL:                jaegerConfig.URL,
 		NamespaceSelector:  jaegerConfig.NamespaceSelector,
 		Integration:        integration,
 		IntegrationMessage: error,
 	}
 
 	return info, http.StatusOK, nil
-}
+}
```

Comment:
@aljesusg build failed apparently due to newline

Reply:
Fixed